features/unsupported/USBHost/USBHostMSD/USBHostMSD.cpp

```diff
@@ -29,13 +28,16 @@
 #define GET_MAX_LUN             (0xFE)
 #define BO_MASS_STORAGE_RESET   (0xFF)
 
-USBHostMSD::USBHostMSD(const char * rootdir) : FATFileSystem(rootdir)
+USBHostMSD::USBHostMSD()
```

Comment:
for changes like this, please note in the commit and in the PR description that this is a breaking change?